scripts/furuno_fetch.py

```
@@ -54,7 +54,12 @@ def main(radar: str):
     if not files:
         return
     os.chdir(LOCAL + radar)
-    subprocess.call(["/home/meteor_ldm/pyWWA/util/gr.csh", radar])
+    # We don't care what happens with this command
+    subprocess.call(
```

Ruff (0.15.21)

[error] `subprocess` call: check for execution of untrusted input

(S603)